website/content/installation/_index.md

```diff
@@ -145,6 +155,38 @@ configInline:
      - 198.51.100.0/24
 ```
 
+{{% notice note %}}
+If you want to deploy metallb using the [experimental FRR mode](https://metallb.universe.tf/configuration/#enabling-bfd-support-for-bgp-sessions), the following value must be set:
+
+```yaml
+speaker:
+  frr:
+    enabled:true
+```
+
+{{% /notice %}}
+
+## Using the MetalLB Operator
+
+The MetalLB Operator is available on OperatorHub. It allows configuring MetalLB via CRDs (although [it's planned to have the configuration being handled directly by MetalLB](https://github.com/metallb/metallb/pull/942)).
```

Comment:
It might be irrelevant or discouraging to using the MetalLB Operator mentioning that the MetalLB community is discussing (not officially approved yet) a move of CRDs to the MetalLB repository.

```suggestion
The MetalLB Operator is available on [OperatorHub](https://operatorhub.io/operator/metallb-operator). It eases the deployment and life-cycle of MetalLB in a cluster and allows configuring MetalLB via CRDs.
```